### PR TITLE
doc: update faq

### PR DESCRIPTION
**Pull request by markus2330**
@mpranj if it compiles and it is easy for you, please add to release.

## Basics

- [ ] Short descriptions of your changes are in the release notes
      (added as entry in `doc/news/_preparation_next_release.md` which
      contains `_(my name)_`)
      **Please always add something to the release notes.**
- [ ] Details of what you changed are in commit messages
      (first line should have `module: short statement` syntax)
- [ ] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildservers are happy. If not, fix **in this order**:
  - [ ] add a line in `doc/news/_preparation_next_release.md`
  - [ ] reformat the code with `scripts/dev/reformat-all`
  - [ ] make all unit tests pass
  - [ ] fix all memleaks
- [ ] The PR is rebased with current master.

## Checklist

- [ ] I added unit tests for my code
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I added code comments, logging, and assertions as appropriate (see [Coding Guidelines](https://master.libelektra.org/doc/CODING.md))
- [ ] I updated all meta data (e.g. README.md of plugins and [METADATA.ini](https://master.libelektra.org/doc/METADATA.ini))
- [ ] I mentioned [every code](/.reuse/dep5) not directly written by me in [reuse syntax](https://reuse.software/)

## Review

- [ ] Documentation is introductory, concise, good to read and describes everything what the PR does
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to [our Coding Guidelines](https://master.libelektra.org/doc/CODING.md)
- [ ] APIs are conforming to [our Design Guidelines](https://master.libelektra.org/doc/DESIGN.md)
- [ ] Code is consistent to [our Design Decisions](https://master.libelektra.org/doc/decisions)

## Labels

- [ ] Add the "work in progress" label if you do not want the PR to be reviewed yet.
- [ ] Add the "ready to merge" label **if the basics are fulfilled** and no further pushes are planned by you.

### CODE REVIEW

**Comment by markus2330** [comment]
I see that 0.9.9 already happened *fireworks*

So let us merge this afterwards :wink:

File: doc/news/_preparation_next_release.md [hunk 1]
```diff
@@ -241,6 +241,56 @@ you up-to-date with the multi-language support provided by Elektra.
...
+- Added screenshots and a quick walk through in the Qt-GUI README. _(@muskater)_ _(@lawli3t)_ _(Aydan Ghazani @4ydan)_
+- Improve example for kdb-restore man page and fix typos _(@JakobWonisch)_
+- Improve example for kdb-restore man page and fix typos _(@JakobWonisch)_
+- Fix some typos in the "Getting Started" page _(Ivaylo Ivanov)_
+- Added some improvements to the core api documentation _(@muskater)_
+- Add screenshots with hints to CLion PR tutorial _(@JakobWonisch)_
+- Fix typo in elektra-backends man page _(@JakobWonisch)_
+- Expanded the webside guid for easier understanding and linked to cmake.org. _(Philipp Nirnberger @nirnberger)_
+- Expanded the webside guide for easier understanding and linked to cmake.org. _(Philipp Nirnberger @nirnberger)_
+- Fix small error in CLion tutorial: CMake options would create a directory named `~` in home directory _(Maximilian Irlinger @atmaxinger)_
```
Comment (mpranj) [inline doc/news/_preparation_next_release.md:293]:
```suggestion
```

Comment (mpranj) [inline doc/news/_preparation_next_release.md:293]:
```suggestion
- Update [FAQ](/doc/help/elektra-faq.md). _(Markus Raab)_
```

**Comment by mpranj** [comment]
Thank you for updating the FAQ. 
Sorry, I already hit the "publish release" button shortly before I saw this PR.

**Comment by mpranj** [comment]
Thank you!

**Pull request merged**